### ISSUE TITLE
False positive on `Layout/MultilineMethodCallIndentation` cop

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#12226]
## Expected behavior

No changes.

#### Before

```ruby

def banner_ids_params
  params[:included].find do |data|
    data[:type] == 'ads'
  end&.dig('relationships', 'banners', 'data')
     &.map { |param| Integer(param['id']) }
end
```

#### After

```ruby

def banner_ids_params
  params[:included].find do |data|
    data[:type] == 'ads'
  end&.dig('relationships', 'banners', 'data')
                   &.map { |param| Integer(param['id']) }
end
```

## RuboCop version

```bash
ydakuka@yauhenid:~/Work/project$ bin/rails_docker rubocop -V
1.56.4 (using Parser 3.2.2.3, rubocop-ast 1.29.0, running on ruby 2.7.8) [x86_64-linux]
  - rubocop-capybara 2.19.0
  - rubocop-factory_bot 2.24.0
  - rubocop-performance 1.19.1
  - rubocop-rails 2.21.1
  - rubocop-rake 0.6.0
  - rubocop-rspec 2.24.0
  - rubocop-thread_safety 0.5.1
```
